### PR TITLE
Migrate from WebSocket to LiveView with infinite scrolling

### LINKED ISSUE #7: Explore LiveView to implement inifinite scrolling
Reference: https://hexdocs.pm/phoenix_live_view/bindings.html#scroll-events-and-infinite-stream-pagination

### PR DESCRIPTION
**Pull request by ariefrahmansyah**
Close #7 

The current implementation of the web server is using WebSocket which opens once the page is loaded and starts sending the data in chunks. In other words, the web server is always sending the whole data to the user even though they are not viewing it (scrolling down).

The new approach is using Phoenix LiveView + Infinite Scrolling as described here: https://hexdocs.pm/phoenix_live_view/bindings.html#scroll-events-and-infinite-stream-pagination
The performance improvement is noticeable as we are only loading the necessary data at the beginning.

Before:

https://github.com/ariefrahmansyah/remembergaza/assets/8122852/602145c9-5cae-4a79-95a6-1ea8c3809e7e

After:

https://github.com/ariefrahmansyah/remembergaza/assets/8122852/0126394c-7815-462b-ab45-0382744ed816

### CODE REVIEW

**Comment by AsadSaleh** [comment]
Wow, so cool!

Can't wait to get this merged.

File: src/lib/remembergaza_web/live/home_live.html.heex [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    phx-viewport-bottom="next-page"
+    phx-page-loading
+    class="grid gap-6 md:grid-cols-3 mb-8"
+  >
+    <div :for={{_, victim} <- @streams.victims}>
+      <h3><%= victim["name"] %></h3>
+      <p><%= victim["en_name"] %></p>
+      <%= if victim["age"] != nil and victim["age"] != -1 do %>
+        <p class="text-sm text-slate-500"><%= victim["age"] %> years old</p>
+      <% end %>
```
Comment (ariefrahmansyah) [inline src/lib/remembergaza_web/live/home_live.html.heex:29]: I've ported your changes here @AsadSaleh
The difference is we do not display the "unknown age" anymore:

image

Comment (AsadSaleh) [reply src/lib/remembergaza_web/live/home_live.html.heex:29]: Hmm.., will there be a height inconsistency because of it?

Comment (AsadSaleh) [reply src/lib/remembergaza_web/live/home_live.html.heex:29]: After looking at it again, I think the height is all consistent.

**Review by AsadSaleh — COMMENTED** [review]
Looks good to me!

**Pull request merged**